openid-sse-framework-1_0.txt

```diff
@@ -1681,13 +1967,14 @@ Tulshibagwale, et al.   Expires December 10, 2021              [Page 30]
    Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
 
    {
+     "stream_id": "f67e39a0a4d34d56b3aa1bc4cff0069f",
      "subject": {
        "format": "phone",
        "phone_number": "+1 206 555 0123"
```

Comment:
We should lose the spaces in the phone number format, according to the IETF SecEvents Subject Identifier spec.